public/studio.h

```diff
@@ -1773,7 +1773,7 @@ struct thinModelVertices_t
 	{
 		Assert( ( m_numBoneInfluences  >= 1 ) && ( m_numBoneInfluences  <= 3 ) );
 		Assert( ( boneWeights.numbones >= 1 ) && ( boneWeights.numbones <= m_numBoneInfluences ) );
-		int    numStoredWeights = max( 0, ( m_numBoneInfluences - 1 ) );
+		int    numStoredWeights = MAX( 0, ( m_numBoneInfluences - 1 ) );
```

Comment:
Please avoid using the `MAX` macro and use instead this function https://github.com/danielga/sourcesdk-minimal/blob/a218e990b9d562824675fc2076d8087c5ceb9bc9/public/tier0/basetypes.h#L120-L126 or the [`std::max` function](https://en.cppreference.com/w/cpp/algorithm/max).